src/Interpreters/JIT/compileFunction.h

```diff
@@ -51,10 +52,10 @@ struct AggregateFunctionWithOffset
 };
 
 using JITCreateAggregateStatesFunction = void (*)(AggregateDataPtr);
-using JITAddIntoAggregateStatesFunction = void (*)(ColumnDataRowsSize, ColumnData *, AggregateDataPtr *);
-using JITAddIntoAggregateStatesFunctionSinglePlace = void (*)(ColumnDataRowsSize, ColumnData *, AggregateDataPtr);
+using JITAddIntoAggregateStatesFunction = void (*)(ColumnDataRowsOffset, ColumnDataRowsSize, ColumnData *, AggregateDataPtr *);
```

Comment:
Maybe second parameter should be `ColumnDataRowsOffset` too? This naming makes it feel like signature `foo(type * start, int count)` instead of `foo(type * start, type * end)`. (I know there are no pointers, but it's the simpliest way to explain what I mean).

Reply:
Agree.